motioneye/handlers/movie.py

```diff
@@ -32,6 +32,14 @@ async def get(self, camera_id, op, filename=None):
             camera_id = int(camera_id)
             if camera_id not in config.get_camera_ids():
                 raise HTTPError(404, 'no such camera')
+            # block access to admin-only cameras for non-admin users
+            camera_config = config.get_camera(camera_id)
+            if (
+                camera_config
+                and camera_config.get('@admin_only')
+                and self.current_user != 'admin'
+            ):
+                raise HTTPError(403, 'access denied')
```

Comment:
Same as above, adding more details to this server-side debug log message:
```suggestion
                raise HTTPError(403, f'access denied to admin-only camera "{camera_id}" for operation "{op}"')
```